samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -11,12 +11,12 @@
 #include <map>
 #include <memory>
 #include <mutex>
+#include <openvino/openvino.hpp>
```

Comment:
@vladimir-dudnik looks like idea was to have all ov headers in one place.
At the current moment we mix ov header with system headers.

Reply:
Not clear. If we want all OV headers in one place - why don't OV put all of them into one wrapper header ? Or you do you mean grouping headers in modules by their meanings ?